src/target/cortexm.c

```diff
@@ -405,67 +405,61 @@ bool cortexm_probe(ADIv5_AP_t *ap)
 #endif
 
 	switch (ap->ap_designer) {
-	case AP_DESIGNER_FREESCALE:
+	case JEP106_FREESCALE:
 		PROBE(kinetis_probe);
 		if (ap->ap_partno == 0x88c) {
 			t->driver = "MIMXRT10xx(no flash)";
 			target_halt_resume(t, 0);
 		}
 		break;
-	case AP_DESIGNER_CS:
```

Comment:
Please clarify what this case was and how it's now handled

Reply:
the context of this removal is [here](https://github.com/blackmagic-debug/blackmagic/blob/f16fe54d24c3278378a8651f7e777dd5ad0f0c13/src/target/adiv5.h#L159-L163)

now this "AP Designer" is aliased to STM, and the case that handles these devices follows

Reply:
sorry, this was supposed to direct you to the correct lines in the commit, but github did not collaborate, follow the lines in the link